### PR TITLE
DRIVERS-1878: Tests for client-side errors with snapshot reads use incorrect syntax for test.expectEvents

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/DRIVERS-1878

### CODE REVIEW

**Review by alcaeus — COMMENTED** [review]
Changes LGTM.

Can we bump the patch version for the sessions spec to note that we updated tests? Similar with unified test format, we should bump the patch version and add a changelog entry to point out the clarification.

**Review by ShaneHarvey — APPROVED** [review]
LGTM

**Review by alcaeus — APPROVED** [review]

**Comment by jmikola** [comment]
@dariakp Would you mind reviewing this on behalf of the session spec owners? I'm not sure why GitHub did not automatically tag you as it did with @rstam.

**Review by dariakp — APPROVED** [review]

File: source/sessions/snapshot-sessions.rst [hunk 1]
```diff
@@ -3,13 +3,13 @@ Snapshot Reads Specification
...
 :Spec Title: Snapshot Reads Specification
-:Spec Version: 1.2.1
+:Spec Version: 1.2.2
 :Author: Boris Dogadov
 :Advisors: Jeff Yemin, A. Jesse Jiryu Davis, Judah Schvimer
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 5.0
-:Last Modified: 16-Jul-2021
+:Last Modified: 06-Aug-2021
```
Comment (jmikola) [inline source/sessions/snapshot-sessions.rst:12]:
```suggestion
:Last Modified: 09-Aug-2021
```

File: source/sessions/snapshot-sessions.rst [hunk 2]
```diff
@@ -289,3 +289,4 @@ Changelog
 :2021-06-28: Raise client side error on < 5.0.
 :2021-06-29: Send readConcern with all snapshot session commands.
 :2021-07-16: Grammar revisions. Change SHOULD to MUST for startTransaction error to comply with existing tests.
+:2021-08-06: Updated client-side error spec tests to use correct syntax for ``test.expectEvents``
```
Comment (jmikola) [inline source/sessions/snapshot-sessions.rst:292]:
```suggestion
:2021-08-09: Updated client-side error spec tests to use correct syntax for ``test.expectEvents``
```

File: source/unified-test-format/unified-test-format.rst [hunk 3]
```diff
@@ -3,13 +3,13 @@ Unified Test Format
...
 :Spec Title: Unified Test Format
-:Spec Version: 1.5.3
+:Spec Version: 1.5.4
 :Author: Jeremy Mikola
 :Advisors: Prashant Mital, Isabel Atkinson, Thomas Reggi
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: N/A
-:Last Modified: 2021-07-29
+:Last Modified: 2021-08-06
```
Comment (jmikola) [inline source/unified-test-format/unified-test-format.rst:12]:
```suggestion
:Last Modified: 2021-08-09
```

File: source/unified-test-format/unified-test-format.rst [hunk 4]
```diff
@@ -3251,6 +3251,9 @@ spec changes developed in parallel or during the same release cycle.
 Change Log
 ==========
 
+:2021-08-06: Updated all existing schema files to require at least one element
```
Comment (jmikola) [inline source/unified-test-format/unified-test-format.rst:3254]:
```suggestion
:2021-08-09: Updated all existing schema files to require at least one element
```

**Pull request merged**